src/schema.js

```diff
@@ -86,8 +86,21 @@ const schema = {
     },
     list_item: {
       parent: [{ type: 'ol_list' }, { type: 'ul_list' }],
-      nodes: [{ match: [{ object: 'text' }, { type: 'link' }] }],
-      marks: [{ type: 'bold' }, { type: 'italic' }, { type: 'code' }],
+      nodes: [
+        {
+          match: [
+            { object: 'text' },
+            { type: 'link' },
+            { type: 'paragraph' },
+            { type: 'softbreak' }
```

Comment:
I don't think it is valid to just have a soft break as a list item. A soft break should be inside a paragraph.

Reply:
Unsure how to implement this. Currently, it is parsing correctly. If I remove `'softbreak'` from `list_item` and rely on `list_item` being a `'paragraph'`, which can be a `'softbreak'`, it does not parse correctly.